BigriverBookstore-api/BigriverBookstore-api.Server.WebService/BookRepository.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using BigriverBookstore_api.ServiceModel;
+
+
+namespace BigriverBookstore_api.WebService
+{
+
+    public static class BookRepository
```

Comment:
In order to be consistent with the directory structure choose I would though that repositories should have been namespaced under `Repositories`, same as `Controllers` and `Configurations`

Not that this is the best way to separate them, I would argue that separating by features is better then by concepts.